test/functional/test-3p-frame.js

```diff
@@ -120,9 +121,13 @@ describe('3p-frame', () => {
       div.setAttribute('data-ping', 'pong');
       div.setAttribute('width', '50');
       div.setAttribute('height', '100');
+      div.setAttribute('layout', 'responsive');
 
       const width = window.innerWidth;
       const height = window.innerHeight;
+      div.getLayout = function() {
```

Comment:
Yes given the div element is not loaded via "normal" flow and getLayout is required